docs/release-notes/dont-accept-epoch-bad-values.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Improvements**
+
+-  Trials: Reporting a training or validation metric with 'epoch' set with a non numeric value will
+   now return an error.
```

Comment:
the second "with" is confusing

maybe this?

-  Trials: Reporting a training or validation metric with 'epoch' set to a non-numeric value will now return an error.